packages/static-hosting/lib/handlers/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "lib": ["es2019"],
+    "removeComments": true,
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "sourceMap": true,
+    "target": "es2017",
```

Comment:
should the target and lib match? i've no clue

Reply:
They don't _need_ to match. My understanding is that lib is what you're using when writing TS whereas target is what version of JS it transpiles to.